prov/verbs/src/verbs_eq.c

```diff
@@ -164,6 +165,9 @@ fi_ibv_eq_cm_process_event(struct fi_ibv_eq *eq, struct rdma_cm_event *cma_event
 	case RDMA_CM_EVENT_ESTABLISHED:
 		*event = FI_CONNECTED;
 		entry->info = NULL;
+		ret = fi_ibv_modify_qp(cma_event->id->qp);
```

Comment:
Yes. Since we can change the QP attributes after the connection is established we can do it in one place. I believe verbs/RDM may not need this change as it has unexpected queue to handle incoming sends that don't have posted recvs.